### PR TITLE
Update readme after audit fixes

### PR DESCRIPTION
**Pull request by bb111189**
Update readme for ssnlist, proxy and multisig wallet
1. addssn
2. addssn after upgrade
3. combine update min, max and contract max stake into update contract parameter
4. add drain proxy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,10 +206,9 @@ These transitions are meant to redirect calls to the corresponding `SSNList` con
...
-|`update_contractmaxstake (max_stake : Uint128)` | `update_contractmaxstake (max_stake : Uint128, initiator : ByStr20)`|
-|`add_ssn (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128)` | `add_ssn (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128, initiator : ByStr20)`|
+|`update_staking_parameter (min_stake : Uint128, max_stake : Uint128, contract_max_stake : Uint128)` | `update_staking_parameter (min_stake : Uint128, max_stake : Uint128, contract_max_stake : Uint128, initiator : ByStr20)`|
+|`add_ssn (ssnaddr : ByStr20, urlraw : String, urlapi : String)` | `add_ssn (ssnaddr : ByStr20, urlraw : String, urlapi : String, initiator : ByStr20)`|
+|`add_ssn_after_upgrade (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128)` | `add_ssn (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128, initiator : ByStr20)`|
```
Comment (sandipbhoir) [inline README.md:211]: seems typo, later should be `add_ssn_upgrade` call in ssnlist contract?

Comment (bb111189) [reply README.md:211]: https://github.com/Zilliqa/staking-contract/pull/33/files

Seems like is `add_ssn_after_upgrade`

Comment (sandipbhoir) [reply README.md:211]: second column on line 211 should be `add_ss_after_upgrade` ?

Comment (bb111189) [inline README.md:211]:
```suggestion
|`add_ssn_after_upgrade (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128)` | `add_ssn_after_upgrade (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128, initiator : ByStr20)`|
```

**Pull request merged**